index.coffee

```diff
@@ -194,6 +71,7 @@ initPassport = ->
     clientSecret: opts.FACEBOOK_SECRET
     callbackURL: "#{opts.APP_URL}#{opts.facebookCallbackPath}"
     passReqToCallback: true
+    state: true
```

Comment:
This is the important line that uses passport's, undocumented, but still first-class CSRF support https://github.com/jaredhanson/passport-facebook/issues/14